src/modules/UI/components/Icon/Icon.ui.js

```diff
@@ -14,7 +14,7 @@ import * as Constants from '../../../../constants/indexConstants'
 type Props = {
   style: StyleSheet.Styles,
   name: string,
-  size: number,
+  size?: number,
```

Comment:
This is not going to break this for now, but how are you dealing with size - is there now a default? are handling it with the style?